server/server.js

```diff
@@ -161,11 +162,36 @@ module.exports = function(port, db, githubAuthoriser) {
                     contents: message,
                     timestamp: timestamp
                 }).then(function(result) {
+                    // Update conversation timestamp
                     conversations.updateOne({
                         _id: conversationID
                     }, {
                         $set: {lastTimestamp: timestamp}
                     });
+                    // Upsert a "new_messages" notification for each participant
```

Comment:
It's a flag for update operations in MongoDB that says to insert a new document with the update body if the document to be updated doesn't already exist; I've changed the comment to clarify this functionality.